{{cookiecutter.project_shortname}}/site/setup.py

```diff
@@ -0,0 +1,3 @@
+from setuptools import setup
```

Comment:
It's not possible to only use setup.cfg?

Reply:
no, if we want to install it as editable via Pipfile... it requires a setup.py...